lib/webmachine.ml

```diff
@@ -827,7 +827,7 @@ module Make(IO:IO) = struct
         >>~ fun l_mod ->
             match (u_mod, l_mod) with
             | (Some l_mod', Some u_mod') ->
-               (match (Util.Date.parse_rfc1123_date_exn l_mod') > (Util.Date.parse_rfc1123_date_exn u_mod') with
+               (match (DATE.parse_rfc1123_date_exn l_mod') > (DATE.parse_rfc1123_date_exn u_mod') with
```

Comment:
Should this be `Ptime.compare x y > 0`?

Reply:
Or rather, `DATE.compare`. It's probably error-prone to rely on polymoprhic compare for a user-provided abstract type.